pkg/operator/starter.go

```diff
@@ -132,6 +132,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 			"v4.1.0/kube-apiserver/localhost-recovery-sa.yaml",
 			"v4.1.0/kube-apiserver/localhost-recovery-token.yaml",
 			"v4.1.0/kube-apiserver/audit-policies-cm.yaml",
+			"v4.1.0/kube-apiserver/apiserver.openshift.io_deprecatedapirequests.yaml",
 		},
 		(&resourceapply.ClientHolder{}).WithKubernetes(kubeClient),
```

Comment:
don't you need to add the apiextensions client?

Reply:
Done.